File: Controllers/ClockProps.cs
namespace time.Controllers;

public class ClockProps {
  public Guid Id {get; set;} = Guid.NewGuid();
  public string Title {get; set;} = "The Time of Your Life";
  public string FontFamily {get; set;} = "courier";
  public int TitleFontSize {get; set;} = 64;
  public int ClockFontSize {get ; set;} = 48;
  public string TitleFontColor {get; set;} = "black";
  public string ClockFontColor {get; set;} = "black";
  public bool BlinkColons {get; set;} = true;
}

File: ClientApp/src/components/ClockProps.js
class ClockProps {
  fontFamily = 'courier'
  availableFontSizes = [12, 24, 48, 64]
  titleFontSize = 64
  clockFontSize = 48
  blinkColons = true
  fontColor = 'black'

  props = {
    fontFamily: this.fontFamily,
    titleFontSize: this.titleFontSize,
    clockFontSize: this.clockFontSize,
    blinkColons: this.blinkColons,
    availableFontSizes: this.availableFontSizes,
    fontColor: this.fontColor,
  }
}

export default ClockProps
